app/models/manageiq/providers/regions.rb

```diff
@@ -0,0 +1,46 @@
+module ManageIQ::Providers
+  class Regions
+    class << self
+      def regions
+        from_file.merge(additional_regions).except(*disabled_regions)
+      end
+
+      def all
+        regions.values
+      end
+
+      def names
+        regions.keys
+      end
+
+      def find_by_name(name)
```

Comment:
This is used by azure and amazon but I'm not sure that it buys us much over just `.regions[name]` unless a subclass had to override it

Reply:
Yeha, and it also triggers invalid rubocops where they say to stop using `find_by`